src/qos.ts

```diff
@@ -62,6 +62,8 @@ export const startQosStatsCollection = (session: WebPhoneSession): void => {
 
     session.on('terminated', function() {
         previousGetStatsResult && previousGetStatsResult.nomore();
+        session.logger.log('Release media streams');
+        session && session.mediaStreams.release();
```

Comment:
No need to check `session` here, you access `session.logger.log` 1 line above, or check there too

Reply:
should check if session.mediaStreams exists